apps/session-manager/src/services/lollipop.ts

```diff
@@ -41,13 +44,14 @@ export type GenerateLCParamsErrors =
 export const generateLCParams: (
   assertionRef: AssertionRef,
   operationId: NonEmptyString,
+  fiscalCode?: FiscalCode,
```

Comment:
When this param can be undefined?

Reply:
during `fast-login` which calls the `expressLollipopMiddleware`